src/lib/atoms/TagLink/README.md

```diff
@@ -0,0 +1,3 @@
+# TagLink
```

Comment:
Same for this readme, it should tell how this component will be used, not how it's built